dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -529,10 +526,20 @@ Configuration InstallCAM
 				$CatalinaHomeLocation = $using:CatalinaHomeLocation
 				$catalinaBase = "$CatalinaHomeLocation" #\$using:AUIServiceName"
 
-                Write-Verbose "Install Nuget and AzureRM packages"
+                Write-Verbose "Ensure Nuget Package Provider and AzureRM module are installed"
 
-				Install-packageProvider -Name NuGet -Force
-				Install-Module -Name AzureRM -Force
+				If(-not [bool](Get-PackageProvider -ListAvailable | where {$_.Name -eq "NuGet"}))
```

Comment:
I got the 'no AzureRM module' error again. Clearly there is a module and it seems to already be installed so the most conservative option seemed to be to just pull the package provider and module if needed. We'll have to see if it fixes things. The best reason I can come up with for the intermittent failure is that NuGet is still getting configured when the InstallModule is called so it fails strangely.